content/api/commands/log.md

```diff
@@ -33,8 +33,7 @@ to the number of arguments.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`cy.log()` yields `null`.</li><li>`cy.log()` cannot be chained
-further.</li></List>
+- `cy.log()` yields `null`.
```

Comment:
Hm, this is a really good question. `cy.log` is "cy.wrap(), but it yields null and has a different name." This keeps confusing me - I always expect it to log the subject, rather than log its arguments, but that's not what it's used for.

Thinking about it a bit more, .log() should probably be a command in the code rather than a query. I'll open a PR to change it back to a command.